docs/_quarto.yml

```diff
@@ -0,0 +1,106 @@
+project:
+  type: website
+  pre-render: version_config.py
+  output-dir: _site
+
+website:
+  title: "Vetiver (Python)"
+  description: "Version, share, deploy, and monitor your Python models"
+  repo-url: https://github.com/rstudio/vetiver-python
+  repo-actions: [edit, issue]
+  page-navigation: true
+  favicon: "figures/logo.svg"
+  page-footer:
+    left: |
+      Proudly supported by
+      [![](https://www.rstudio.com/assets/img/posit-logo-fullcolor-TM.svg){fig-alt="Posit PBC" width=65px}](https://posit.co/)
+    center:
+      - text: "Main site"
+        href: https://vetiver.rstudio.com
+    right:
+      - icon: github
+        href: https://github.com/rstudio/vetiver-python
+        aria-label: Vetiver python GitHub
+  navbar:
+    background: primary
+    pinned: true
+    logo: "figures/logo.svg"
+    logo-alt: "vetiver-home"
+    search: true
+    left:
+      - text: "Reference"
+        file: reference/index.qmd
+      - text: "Advanced Usage"
+        menu:
+          - custom_handler.md
+      - text: "Changelog"
+        file: changelog.md
+      - text: "Vetiver main site"
+        href: https://vetiver.rstudio.com
+
+  sidebar:
+    - id: reference
+      contents: "_sidebar.yml"
+
+quartodoc:
+  style: pkgdown
+  title: "Vetiver-{{< env VERSION >}}"
```

Comment:
For whatever reason, any spaces between `vetiver` and `{{< env VERSION >}}` in this title causes the version to not show up in the sidebar. If I write `{{< env VERSION >}} vetiver` it does show, or if I have a title with no blank spaces.